src/server/game.js

```diff
@@ -227,11 +227,20 @@ let game = function() {
 		}
 	};
 
+	let getInitialDataPayload = function() {
```

Comment:
Moving this function should be reverted once the game-network circular dependency is resolved.